lib/src/page_list_viewport_gestures.dart

```diff
@@ -442,7 +636,29 @@ class DeprecatedPanAndScaleVelocityTracker {
       return;
     }
 
+    // Check that the two swipes consequtively considered for repeated
+    // swiping acceleration are collinear
+    if (_isPossibleRepeatedAcceleratedSwipe && !((_previousLaunchVelocity.dy * velocity.dy).isNegative)) {
```

Comment:
Is the multiplication and `isNegative` check about comparing direction? Are you just looking for opposite signs? If so, can you use the `sign` property of `num`s, instead?

Also, is there a reason that they only checks vertical motion?